02stl/cpp/particlesMMap.cc

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <map>
+#include <string>
+#include <iostream>
+
+int main()
+{
+  FILE* ifp = fopen("02stl/cpp/particleList.txt","r");
```

Comment:
naked pointers? file that don't automatically cleanup when out of scope?
That's the kind of bad habits we would rather they not use.
Why not use STL's for this as well?
We are not expecting that they should no C and fopen, so there is no cost.